lib/ore/aes128.rb

```diff
@@ -71,6 +71,11 @@ def encrypt(plaintext)
         encrypt_bool(plaintext)
       when Range
         encrypt_range(plaintext)
+      when Date
+        coerced = plaintext - Date.new(1970, 1, 1)
```

Comment:
It looks like the result of subtracting two `Date`s is a `Rational`:

```ruby
>> (Date.new(1960, 1, 1) - Date.new(1970, 1, 1)).class
=> Rational
```

Which `encrypt_u64` (in the Rust code, at least) is probably going to lose its lunch over, because it expects an `Integer`.

Reply:
Oh yes, oops. That is supposed to have a `to_i` on the end.